introduction.md

```diff
@@ -0,0 +1 @@
+Hi I'm Brock. Looking forward to doing the Mastering GitHub tutorial on Code School.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.